docker/fetch_tezos_sources.py

```diff
@@ -45,13 +45,7 @@
     .strip()
 )
 
-os.chdir("opam-repository-tezos")
+os.chdir("opam-repository")
 subprocess.run(["git", "checkout", opam_repository_tag])
 subprocess.run(["rm", "-rf", ".git"])
-subprocess.run(["rm", "-r", "zcash-params"])
-subprocess.run(["scripts/create_opam_repo.sh"])
-subprocess.run(["mv", "opam-repository", ".."])
-os.chdir("..")
-subprocess.run(["rm", "-rf", "opam-repository-tezos"])
-os.chdir("opam-repository")
 subprocess.run(["opam", "admin", "cache"])
```

Comment:
Caching an entire `ocaml/opam-repository` is excessive, we should filter it somehow prior to caching